fastlane/lanes/ci_releases.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+PIPELINES_ROOT = 'release-pipelines'
+
+platform :ios do
+  lane :trigger_code_freeze_in_ci do
+    buildkite_trigger_build(
+      buildkite_organization: BUILDKITE_ORGANIZATION,
+      buildkite_pipeline: BUILDKITE_PIPELINE,
+      branch: DEFAULT_BRANCH,
+      pipeline_file: File.join(PIPELINES_ROOT, 'start-code-freeze.yml'),
+      message: 'Start Code Freeze'
+    )
+  end
+
+  lane :trigger_complete_code_freeze_in_ci do |release_version: release_version_current|
+    buildkite_trigger_build(
+      buildkite_organization: BUILDKITE_ORGANIZATION,
+      buildkite_pipeline: BUILDKITE_PIPELINE,
+      branch: release_branch_name(release_version: release_version),
+      pipeline_file: File.join(PIPELINES_ROOT, 'complete-code-freeze.yml'),
+      message: "Complete code freeze for #{release_version}",
+      environment: { RELEASE_VERSION: release_version }
+    )
+  end
+end
```

Comment:
I decided to name this `ci_releases.rb` instead of `release_management_in_ci.rb` like other projects because it makes tab completion easier. 

Otherwise, with `re...` one gets hits for `relaese.rb` and `release_management_in_ci.rb`